Dockerfile

```diff
@@ -39,16 +39,25 @@ ADD . /build-tools-ci
 
 # Collect the components we need for this image
 RUN apt-get update
-RUN apt-get install -y ruby jq curl
+RUN apt-get install -y --allow-unauthenticated ruby jq curl
 RUN gem install circle-cli
 RUN composer -n global require -n "hirak/prestissimo:^0.3"
 
-# Create an unpriviliged testuser
-RUN groupadd -g 999 tester && \
-    useradd -r -m -u 999 -g tester tester && \
-    chown -R tester /usr/local && \
-    chown -R tester /build-tools-ci
-USER tester
+# Update git version.
+RUN apt update -y && \
+    apt install -y make libssl-dev libghc-zlib-dev libcurl4-gnutls-dev libexpat1-dev gettext unzip && \
+    wget https://github.com/git/git/archive/v2.31.1.tar.gz -O git.tar.gz && \
```

Comment:
Could we maybe upgrade git using https://launchpad.net/~git-core/+archive/ubuntu/ppa?

Reply:
Thanks! I just changed it to use ppa instead of compiling from source.